src/main/java/minicraft/item/TileItem.java

```diff
@@ -91,10 +91,10 @@ public boolean interactOn(Tile tile, Level level, int xt, int yt, Player player,
 		
 		String note = "";
 		if (model.contains("WALL")) {
-			note = Localization.getLocalized("minicraft.notification.invalid_placement") + " " + Tiles.getName(validTiles.get(0)) + "!";
+			note = Localization.getLocalized("minicraft.notification.invalid_placement") + " " + Localization.getLocalized(this.validTiles.get(0)) + "!";
 		}
 		else if (model.contains("DOOR")) {
-			note = Localization.getLocalized("minicraft.notification.invalid_placement") + " " + Tiles.getName(validTiles.get(0)) + "!";
+			note = Localization.getLocalized("minicraft.notification.invalid_placement") + " " + Localization.getLocalized(this.validTiles.get(0)) + "!";
 		}
```

Comment:
I did say delete all changes in the file; delete ALL the changes in this file.

Reply:
But it localizes the tile. Otherwise it will say the translated text with the english tile

Reply:
It won't be localized, since the key is wrong, no matter what you do. So just keep it as it is.

Reply:
Wdym the key is wrong? It translates it correctly

Reply:
Then don't translate it. We can fix that issue properly when we get to that point, but that is something I will have to do later, not something you will do now.